### DISCUSSION TITLE
Upgrading to v10 failing for CSS error

### DISCUSSION THREAD

**Discussion by swr7der** [vercel/next.js#19283] [Help]
I'm new to Next.js. I developed the application on v9 with Ant Design. Added the `less` config from the official Nextjs's Github `antd-nextjs` project. It somehow worked for :  less + module.css + global css files with 3rd party css files.

I recently updated the application to v10 and modified the config but till now I could just figured out for (less + module.css) working in the application. The issue is I have to use some 3rd party min.css files too as global files but I can't figure out how that can be done. 

My next.config.js file:

    module.exports = withCSS({
        cssModules: true,
        cssLoaderOptions: {
            importLoaders: 1,
            localIdentName: "[local]___[hash:base64:5]",
        },
        ...withLess({
            lessLoaderOptions: {
                javascriptEnabled: true,
                modifyVars: themeVariables, // make your antd custom effective
            },
            webpack: (config, { isServer }) => {
                if (isServer) {
                    const antStyles = /antd\/.*?\/style.*?/
                    const origExternals = [...config.externals]
                    config.externals = [
                        (context, request, callback) => {
                            if (request.match(antStyles)) return callback()
                            if (typeof origExternals[0] === 'function') {
                                origExternals[0](context, request, callback)
                            } else {
                                callback()
                            }
                        },
                        ...(typeof origExternals[0] === 'function' ? [] : origExternals),
             ]
  
             config.module.rules.unshift({
                 test: antStyles,
                 use: 'null-loader',
             });
          
             // SVGs loading
             config.module.rules.push({
                 test: /\.svg$/,
                 issuer: {
                    test: /\.(js|ts)x?$/,
                 },
                 use: ['@svgr/webpack'],
             });
            }
            return config
          },
       }),
        ...nextConfig,
    });

  Using this config 3rd party css are not working. Please help!

**Comment by dryleaf** [comment]
Any updates on this issue?
